crates/tauri-cli/src/lib.rs

```diff
@@ -359,16 +349,17 @@ impl CommandExt for Command {
     let stdout_lines_ = stdout_lines.clone();
     std::thread::spawn(move || {
       let mut line = String::new();
-      let mut lines = stdout_lines_.lock().unwrap();
       loop {
         line.clear();
         match stdout.read_line(&mut line) {
           Ok(0) => break,
           Ok(_) => {
             log::debug!(action = "stdout"; "{}", line.trim_end());
-            lines.extend(line.as_bytes().to_vec());
+            if let Ok(mut lines) = stdout_lines_.lock() {
+              lines.extend(line.as_bytes());
+            }
```

Comment:
i think we should keep the lock() where it was for performance reasons. the next time we will need a lock is after these threads are stopped.